ruby/ql/lib/codeql/ruby/frameworks/ActionController.qll

```diff
@@ -203,10 +203,7 @@ private module Request {
 
   /** A method call on `request` which returns part or all of the request path. */
   private class PathCall extends RequestInputAccess {
-    PathCall() {
-      this.getMethodName() =
-        ["fullpath", "original_fullpath", "original_url", "url", "path", "filtered_path"]
-    }
+    PathCall() { this.getMethodName() = ["path", "filtered_path"] }
```

Comment:
I don't understand this change - aren't the other methods here at least partially user-controlled?